AnkiDroid/src/main/java/com/ichi2/anki/IntentHandler.kt

```diff
@@ -113,7 +116,21 @@ class IntentHandler : Activity() {
         val importResult = handleFileImport(this, intent)
         // Start DeckPicker if we correctly processed ACTION_VIEW
         if (importResult.isSuccess) {
-            Timber.d("onCreate() import successful")
+            val file = File(intent.data!!.path!!)
+            val fileUri = applicationContext?.let {
+                FileProvider.getUriForFile(
+                    it,
+                    it.applicationContext?.packageName + ".apkgfileprovider",
+                    File(it.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS), file.name)
+                )
+            }
+            try {
+                contentResolver.delete(fileUri!!, null, null)
+                Timber.d("onCreate() import successful and downloaded file deleted ")
+            } catch (e: SecurityException) {
+                Timber.d("onCreate() import successful and cannot delete file $e")
```

Comment:
:thinking: I am struggling here - I think it is more than a `Timber.d` since this is an unexpected situation and breaks the functionality. However, we recover from it - the app can keep on working, so it's not a `Timber.e`. It's either an `.i` or a `.w`. I think `.i` fits better for things that are expected and just good to know, this is unexpected and breaks the feature in this PR, so I suggest `.w`

```suggestion
                Timber.w("onCreate() import successful and cannot delete file $e")
```